R/bfast.R

```diff
@@ -1,3 +1,21 @@
+# bfast - Break Detection in the Seasonal and Trend Component of a Univariate Time
+# Copyright (C) 2019  Jan Verbesselt, Dainius Masiliūnas, Dongdong Kong
```

Comment:
Martin Jung authored the stlplus code here. I think a lot of the attribution to Dongdong actually belongs to Martin, see https://github.com/Martin-Jung/bfast/commit/4efb4ea93a54dadd23b22ce2ab429c122e08978d